yaml/grandmaster-clock-ptp-config.yaml

```diff
@@ -0,0 +1,145 @@
+apiVersion: ptp.openshift.io/v1
+kind: PtpConfig
+metadata:
+  name: grandmaster
+  namespace: openshift-ptp
+spec:
+  profile:
+  - name: "grandmaster"
+    ptp4lOpts: "-2 --summary_interval -4"
+    phc2sysOpts: -r -u 0 -m -O -37 -N 8 -R 16 -s ens2f1 -n 24
+    ptpSchedulingPolicy: SCHED_FIFO
+    ptpSchedulingPriority: 10
+    plugins:
+      e810:
+        enableDefaultConfig: true
+    ts2phcOpts: " "
+    ts2phcConf: |
+      [nmea]
+      ts2phc.master 1
+      [global]
+      use_syslog  0
+      verbose 1
+      logging_level 7
+      ts2phc.pulsewidth 100000000
+      #GNSS module - ls /dev/gnss* -al
+      #use gnss0
+      #cat /dev/gnss0 to find available serial port
```

Comment:
Why are you running #cat /dev/gnss0?

Reply:
@josephdrichard will I just remove it? I wasn't sure why it was there tbh. I tried catting dev/tty1 on my local machine and didn't get any output. Not sure if it is different for gnss device

Reply:
I would put something like this, how they describe finding it:
```
    The receiver is accessible on /dev/gnss<id>. In case of multiple receivers
    in the OS, correct device can be found by enumerating either:
    - /sys/class/net/<eth port>/device/gnss/
    - /sys/class/gnss/gnss<id>/device/
```

Reply:
Added this into the reference table: https://github.com/openshift/openshift-docs/pull/55704/files#diff-aa3c49f9019e7d101022d927d96d8a5008b52798e8352237c4de357a24969013R68